app/Http/Controllers/CategoriesController.php

```diff
@@ -9,7 +9,11 @@ class CategoriesController extends Controller
 {
     public function index()
     {
-        $categories = Category::all();
+        $categories = Category::all()->map(function ($category){
+            if($category->words->count() > 0) {
+                return $category;
+            }
+        })->filter();;
```

Comment:
* double semi colon please remove
* you can make the if condition as short hand `return $category->words->count() > 0 ? $category : null;`

Reply:
Ah. I didn't know this was possible. I'll try this.